cypress/integration/save.cy.js

```diff
@@ -62,18 +62,19 @@ const TEST_INDICATOR_NAMES = TEST_INDICATORS.slice(1, 4).map(
 
 describe('saving an AO', () => {
```

Comment:
the hierarchy is `1 describe -> n describe -> 1 it` (`n` meaning _multiple_). But there's now just a single `it` under the second level `describe`s.
Could be refactored to simply `1 describe -> n it`

Reply:
I was thinking that as well, but wasn't sure if there was a plan for additional test scenarios under each of the sub-describes. Will update 👍